kots-exporter/kots-exporter.sh

```diff
@@ -306,9 +306,13 @@ execute_flyway_migration(){
     || error_exit "Job circle-migrator creation error"
 
     echo "Waiting job/circle-migrator to complete -"
-    (kubectl wait job/circle-migrator --namespace "$namespace" --for condition="complete" --timeout=300s \
-    && echo "++++ DB Migration job is successful." ) \
-    || error_exit "Status wait timeout for job/circle-migrator, Check the Log via - kubectl logs pods -l app=circle-migrator"
+    if (kubectl wait job/circle-migrator --namespace "$namespace" --for condition="complete" --timeout=60s); then
```

Comment:
We need bigger no in timeout as it depends in the data need to be migrated.

Reply:
Thanks @atulsingh0 . I was using 60s for testing and committed the value by accident. I'll restore it back to 300s!